unst_msh_gen/ocn_ww3.py

```diff
@@ -18,18 +20,21 @@
 
 from spacing import *
 
+# Load the configuration file
+config = configparser.ConfigParser()
+config.read('config.ini')
```

Comment:
@AliS-Noaa Thank you so much for creating this config file and providing an example of what is needed in that file!!  Instead of hard-coding the expected config file name/location, can the name/path of the configuration file actually be an input as well?

Reply:
Good point. I will revise this.

Reply:
@AliS-Noaa  any update on this?